mne_bids/tests/test_update.py

```diff
@@ -0,0 +1,108 @@
+"""Test for the MNE BIDS path functions."""
+# Authors: Adam Li <adam2392@gmail.com>
+#
+# License: BSD (3-clause)
```

Comment:
I think the file name and the comment here are too generic

Reply:
I've updated it now. Lmk wyt.